appdev/guides/importing-CH-urldispatcher.rst

```diff
@@ -0,0 +1,154 @@
+==========================================================
+Importing from Content Hub and URLdispatcher
+==========================================================
+
+.. figure:: /_static/images/appdev/guides/importingCHdispatcherimages/01ichu.png
+        :align: center
+
+In the previous guide we have seen a little bit about how Content Hub works. In this guide we will see how URLdispatcher works and how to handle imported data from the Content Hub.
+
+Handle data from the Content Hub
+--------------------------------
+
+.. figure:: /_static/images/appdev/guides/importingCHdispatcherimages/02ichu.png
+        :align: center
+
+        OpenStore app from open-store.io
+
+One of the easiest ways of testing an app, is to send a test click to yourself, on Telegram, and opening that click file with the OpenStore through the Content Hub:
+
+.. code-block:: qml
+
+        Connections {
+            target: ContentHub
+
+            onImportRequested: {
+                var filePath = String(transfer.items[0].url).replace('file://', '')
+                print("Should import file", filePath)
+                var fileName = filePath.split("/").pop();
+                var popup = PopupUtils.open(installQuestion, root, {fileName: fileName});
+                popup.accepted.connect(function() {
+                    contentHubInstallInProgress = true;
+                    PlatformIntegration.clickInstaller.installPackage(filePath)
+                })
+            }
+        }
+
+If we tap on the OpenStore app, it will be opened and it will ask if we want to install the click file. Let’s take a look into `the code <https://github.com/UbuntuOpenStore/openstore-app/blob/master/openstore/Main.qml#L85>`_ to see how it is done:
+
+.. image:: /_static/images/appdev/guides/importingCHdispatcherimages/04ichu.png
+        :align: center
```

Comment:
Any need for the image as a code sample? I'd prefer another block of code to a screenshot...

Good developers copy. Great developers paste.